metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/indexing_master.py

```diff
@@ -162,9 +163,21 @@ def zeppelin_notebook_import(self, env):
 
```

Comment:
Do we have to reg ext this out?  Can't we just pull it out of the dict?

Reply:
In Ambari, the Shiro INI content appears as a template like here:

<img width="1274" alt="zeppelin-config-shiro-ini" src="https://user-images.githubusercontent.com/20395490/35026106-0a038a28-fb6f-11e7-93e3-371e5fa353d0.png">

And this can be fetched using Ambari API as follows:
http://<ambari-node>:8080/api/v1/clusters/cl1/configurations?type=zeppelin-shiro-ini&tag=version1

Which I am fetching in the `status_params.py` as
```
zeppelin_shiro_ini_content = config['configurations']['zeppelin-shiro-ini']['shiro_ini_content']
```

I found it easy to parse this from the content. Could you please clarify how I can use `dict` to get this info?

Reply:
I'm sorry, I did not see that you where getting an ini, that it was a 'blob' in ambari.  You are entirely correct.

Reply:
Cool, thanks.